### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
receivers must be in an array
[status code] 400 [reason phrase] InvalidArgument

### CODE REVIEW

**Comment by sroze** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Example usage in a controller:
                 sprintf('/sms/%s/users/%s/jobs', 'my-service-name', 'my-login'),
                 [
                     'message' => 'Si tu veux me parler, envoie-moi un... fax !',
-                    'receivers' => '+33612345678',
+                    'receivers' => [ '+33612345678' ],
```
Comment (meyerbaptiste) [inline README.md]: Extra spaces here.

Comment (ghost) [inline README.md]: updated. thanks !

Comment (meyerbaptiste) [inline README.md]: Squash your commits and after it's good :+1:

**Comment by meyerbaptiste** [comment]
Please @NicolasPion, can you squash your commits?

**Comment by ghost** [comment]
@meyerbaptiste I try, but as you see i make some mistake.. sorry iam new to Github :'(
Can you help me ?

**Comment by meyerbaptiste** [comment]
Read https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History, in particular the chapter about squashing commits and then force push in your branch.

In your _patch-2_ branch:

``` sh
git rebase -i HEAD~4 
# squash the last 3 commits using the editor
git push -f origin patch-2
```

**Comment by ghost** [comment]
Done !
/me RTFM

**Comment by dunglas** [comment]
Thank you @NicolasPion

**Pull request merged**